README.md

```diff
@@ -4,10 +4,92 @@
 ![unit-tests](https://github.com/goverland-labs/goverland-datasource-snapshot/workflows/unit-tests/badge.svg)
 ![golangci-lint](https://github.com/goverland-labs/goverland-datasource-snapshot/workflows/golangci-lint/badge.svg)
 
+Goverland Datasource Snapshot is a micro service designed to manage and synchronize various data sources for Goverland ecosystem.
+At the moment the focus is on Snapshot API.
+The microservice also leverages technologies such as gRPC and NATS.
+PostgresQL is used as database.
+The application exposes Prometheus metrics.
+
+## Usage
+
+- The application initializes various services and workers to manage proposals, spaces, votes, and messages.
+- It sets up gRPC servers for remote procedure calls and integrates with NATS for event publishing.
+
+## Features
+
+- Snapshot SDK Integration: facilitates interaction with Snapshot services for proposals, spaces, votes, and messages.
+- Event Publishing: utilizes NATS for event-driven architecture.
+- gRPC Services: provides gRPC-based APIs for interacting with the application.
+- Database: Integrates with PostgreSQL for storing and managing data.
+- Metrics and Monitoring: exposes Prometheus metrics for monitoring.
+- Health Checks: Implements health check endpoints to monitor the application's status.
+
+## For Developers
+
+### Pre-requisites
+
+- GoLang 1.21
+- Docker
+
+### Run Locally
+
+1. Clone the repository:
+   ```bash
+   git clone https://github.com/goverland-labs/goverland-datasource-snapshot.git
+   cd goverland-datasource-snapshot
+   ```
+2. Install the required Go dependencies:
+   ```bash
+    go mod tidy
+   ```
+3. Spin up the necessary containers:
+   ```bash
+   docker-compose up -d
+   ```
+4. Run the application. Don't forget to provide your own Snapshot API key:
```

Comment:
Also we should describe the .env.dist file (it's a config for running the app)

Reply:
When I run the app locally via `go run main.go`, it looks like this config is ignored. What is the right way to do it?
Btw that's why I used the following command:
   ```bash
    POSTGRES_DSN="host=localhost port=5432 user=postgres password=postgres dbname=postgres sslmode=disable" \
    SNAPSHOT_API_KEY=YOUR_KEY \
    go run main.go
   ```

Reply:
Moved to different PR